lang/pt_BR/LC_MESSAGES/messages.po

```diff
@@ -12863,563 +12863,563 @@ msgstr "Instalação de módulos"
 
-#~ msgid ""
-#~ "Only services and hosts are taken in account in order to calculate this "
-#~ "status. If you modify a template, it won't tell you the configuration had "
-#~ "changed."
-#~ msgstr ""
-#~ "Apenas serviços e hosts são considerados para calcular esse status. Se você "
-#~ "modificar um modelo, ele não informará que a configuração foi alterada."
+msgid ""
+"Only services and hosts are taken in account in order to calculate this "
+"status. If you modify a template, it won't tell you the configuration had "
+"changed."
+msgstr ""
+"Apenas serviços e hosts são considerados para calcular esse status. Se você "
+"modificar um modelo, ele não informará que a configuração foi alterada."
 
-#~ msgid "Y/m/d"
-#~ msgstr "d/m/Y"
+msgid "Y/m/d"
+msgstr "d/m/Y"
 
-#~ msgid "H:i:s"
-#~ msgstr "H:i:s"
+msgid "H:i:s"
+msgstr "H:i:s"
 
-#~ msgid "Y/m/d G:i"
-#~ msgstr "d/m/Y G:i"
+msgid "Y/m/d G:i"
+msgstr "d/m/Y G:i"
 
-#~ msgid "Right Click here to add an autologin link directly to your bookmarks"
-#~ msgstr ""
-#~ "Clique com o botão direito do mouse para adicionar o link de autenticação "
-#~ "automatica diretamente no seu bookmarks"
+msgid "Right Click here to add an autologin link directly to your bookmarks"
+msgstr ""
+"Clique com o botão direito do mouse para adicionar o link de autenticação "
+"automatica diretamente no seu bookmarks"
 
-#~ msgid "Please define autologin authentication key in your profile."
-#~ msgstr "Por favor, defina uma chave de autenticação automatica no seu perfil."
+msgid "Please define autologin authentication key in your profile."
+msgstr "Por favor, defina uma chave de autenticação automatica no seu perfil."
 
-#~ msgid "Connected Users"
-#~ msgstr "Usuários Conectados"
+msgid "Connected Users"
+msgstr "Usuários Conectados"
 
-#~ msgid "Ping"
-#~ msgstr "Ping"
+msgid "Ping"
+msgstr "Ping"
 
-#~ msgid "Tracert"
-#~ msgstr "Tracert"
+msgid "Tracert"
+msgstr "Tracert"
 
-#~ msgid "Y/m/d H:i"
-#~ msgstr "d/m/Y H:i"
+msgid "Y/m/d H:i"
+msgstr "d/m/Y H:i"
 
-#~ msgid "General options"
-#~ msgstr "Opções Globais"
+msgid "General options"
+msgstr "Opções Globais"
 
-#~ msgid "Job groups"
-#~ msgstr "Grupos de Tarefa"
+msgid "Job groups"
+msgstr "Grupos de Tarefa"
 
-#~ msgid "Jobs"
-#~ msgstr "Tarefas"
+msgid "Jobs"
+msgstr "Tarefas"
 
-#~ msgid "Logos"
-#~ msgstr "Logotipos"
+msgid "Logos"
+msgstr "Logotipos"
 
-#~ msgid "Publication rules"
-#~ msgstr "Regras de Publicação"
+msgid "Publication rules"
+msgstr "Regras de Publicação"
 
-#~ msgid "Report design groups"
-#~ msgstr "Grupos de modelos de relatorio"
+msgid "Report design groups"
+msgstr "Grupos de modelos de relatorio"
 
-#~ msgid "Report designs"
-#~ msgstr "Modelos de Relatorio"
+msgid "Report designs"
+msgstr "Modelos de Relatorio"
 
-#~ msgid "Report view"
-#~ msgstr "Visualização de Relatorio"
+msgid "Report view"
+msgstr "Visualização de Relatorio"
 
-#~ msgid "Trash"
-#~ msgstr "Lixeira"
+msgid "Trash"
+msgstr "Lixeira"
 
 #~ msgid " element(s) found"
```

Comment:
why didn't you re-enable these two lines too ?

Reply:
@sc979 done !

Reply:
+1